### ISSUE TITLE
Removed accounts are not handled from Trading Account

### ISSUE THREAD

**Issue by DayalKS2** [Polkadex-Substrate/Polkadex-Open-Beta#420]
Lets say user has registered a funding account with the trading account. Now user has removed/deleted that account itself from Polkadot.js. But when user visits polkadex, that account is still shown as usable and active under user's Account, Trading Account. User is allowed to make that account to 'Use' or even click Add Fund. When Deposit is clicked, it takes user to a blank screen. When Withdraw is clicked, it takes back user to Account page.

Ideally, similar to how Funding Account section reads active account from Polkadot.js, the Trading Account section should only allow user to interact with active funding account supported trading accounts. This needs to be handled.

onchainaddress:{esp5dCFuAuPi6KD3HzhHWQGjgwiNvyWLp6qJW5EGgo7zkhRBJ}

https://user-images.githubusercontent.com/94037805/201506796-d2177892-a136-4a39-948c-eb4e3fc5f19f.mp4

**Comment by nuel77** [comment]
trading accounts are kept independednt of funding account. suppose you want to trade in a browser which does not have polkadot.js , this should be technically possible ..  hence we are allowing trading accounts to operate even if funding account is not there. but the funding account which created the trading account should be linked to your email. 

Better error handling should be there for actions that require the funding account signature though..will look into this.

**Comment by DayalKS2** [comment]
Yes error handling can solve this issue I guess. If you look ta below screen, this is how the DEPOSIT transaction from a trading account with a funding account (Removed/Deleted account) looks like. These need to be handled.
![image](https://user-images.githubusercontent.com/94037805/201525874-7d24125d-e5ef-4769-9c5f-27b0396192f9.png)

**Comment by felixfaisal** [comment]
Normal
